ppr-ui/src/views/mhrSearch/ConfirmMHRSearch.vue

```diff
@@ -195,7 +196,12 @@ export default class ConfirmMHRSearch extends Vue {
 
   private get feeQuantity (): number {
     // Return selected quantity that is not a combination search
-    return this.getSelectedManufacturedHomes.filter(result => result.selected && !result.includeLienInfo).length
+    // return this.getSelectedManufacturedHomes.filter(result => result.selected && !result.includeLienInfo).length
```

Comment:
intentionally left? If so, leave a comment why perhaps

Reply:
@cameron-eyds I refactored to use this commented code in my PR. We will use this section from my changes

Reply:
Gotcha, thank you!